wolfssl/wolfcrypt/error-crypt.h

```diff
@@ -237,8 +237,15 @@ enum {
     SAKKE_VERIFY_FAIL_E = -276,  /* SAKKE derivation verification error */
     MISSING_IV          = -277,  /* IV was not set */
     MISSING_KEY         = -278,  /* Key was not set */
-
-    WC_LAST_E           = -278,  /* Update this to indicate last error */
+    ECDSA_KAT_FIPS_E    = -279,  /* ECDSA KAT failure */
```

Comment:
Agreed.

Reply:
I renamed the PCT error codes so they aren't FIPS specific, since the options that (will) enable them isn't FIPS specific. The other error codes are for FIPS specific tests.